src/global.scss

```diff
@@ -25,6 +25,349 @@
+      }
+    }
+  }
+}
+ion-col {
+  padding: 0;
+}
+.ion-right-col {
+  text-align: end;
+}
+.ion-center-col {
+  text-align: center;
+}
+.ion-left-col {
+  text-align: start;
+}
+.ion-continue-button {
+  width: 125px;
+}
+ion-button {
+  margin: 0;
+}
+.finger-signature-footer {
+  padding-bottom: 0;
+}
+.iontoolbar {
+  padding-left: 20px;
+
+  p {
+    font-size: 13px;
+    color: #00000080;
+    font-weight: 700;
+  }
+}
+.temporary-finished-btn {
+  width: 103px;
+}
+.request-access-btn {
+  width: 115px;
+}
+.try-again {
+  width: 125px;
+}
+// ---------------end-footer-sections-----------------------
+// -----------------error-pages---------------------------
+.error-page {
+  h3 {
+    padding-top: 45px;
+    padding-bottom: 45px;
+  }
+}
+
+// ---------------ion-content-text-css--------------------
+
+.div-text-positions {
+  padding-top: 15px;
+  padding-bottom: 10px;
+}
+// --------------------------h1-tag-css-----------------
+h1 {
+  font-size: 34px;
+  color: #c32032;
+  font-family: 'museo-1000';
+}
+// --------------------------h2-tag-css-----------------
+h2 {
+  font-size: 24px;
+  color: #c32032;
+  font-family: 'museo-700';
+}
+// --------------------------h3-tag-css-----------------
+h3 {
+  font-size: 18px;
+  color: #232c67;
+  font-weight: bold;
+  font-family: 'Trebuchet-bold';
+}
+// --------------------------h4-tag-css-----------------
+h4 {
+  font-size: 18px;
+  color: #807f74;
+
+  font-family: 'Trebuchet_MS';
+}
+// --------------------------h5-tag-css-----------------
+h5 {
+  font-size: 12px;
+  color: #232c67;
+  font-family: 'Trebuchet_MS';
+}
+// -------------------------div-tag-css-----------------
+div {
+  font-size: 16px;
+  color: #7e7e7a;
+  font-family: 'Trebuchet-italic';
+}
+// ------------------------h3-button-css-------------
+.h3-btn {
+  font-size: 18px;
+  color: #ffffff;
+  font-weight: bold;
+  font-family: 'Trebuchet-bold';
+}
+a {
+  background-color: transparent;
+  color: #232c67;
+}
+
+// ---------------------------intro-page-css-------------------------------
+ion-slides {
+  height: 100%;
+  background: rgb(37, 170, 226);
+  background: linear-gradient(0deg, rgba(37, 170, 226, 1) 0%, rgba(37, 170, 226, 1) 50%, rgba(13, 118, 188, 1) 100%);
+}
+// --------------------------ballot-page-css---------------------------
+.mt60 {
+  margin-top: 60px;
+}
+.ion-text-center {
+  text-align: center !important;
+}
+
+ion-input {
+  border: 1px solid lightgray;
+  border-radius: 4px;
+  --padding-start: 10px !important;
+  margin-top: 8px;
+  --font-family: 'Trebuchet_MS';
+}
+
+.err-messages {
+  margin-left: 20px;
+
+  .error {
+    color: var(--ion-color-danger);
+    padding: 0;
+    font-size: 13px;
+    margin-top: 4px;
+  }
+}
+
+.mt30 {
+  margin-top: 30px;
+}
+
+.mt100 {
+  margin-top: 100px;
+}
+
+.main-btn-style {
+  --border-radius: 0;
+  height: 50px;
+  width: 180px;
+  font-size: 20px;
+}
+// ------------------------------------ballot-complete-page----------------------------
```

Comment:
I see that this "global" css file is broken up into different pages. Rather than placing everything at the global level, it's generally best to put the "page-specific" styles into the css file for each page. That keeps the global styles at a minimum, and allows you to scope your styles to the pages that they affect.

Reply:
Everything I read here makes sense to me. Worth mentioning that global.scss is a very small (<100 line) file, which is great because the amount of custom styling which needs to be global is very minimal. That means the surface area of what we're talking about here is small.

I mentioned this idea to Anne over the phone awhile back, but I think parametrizing per-state builds (e.g., an app for Virginia, an app for California, and app for Hawaii, etc) can probably be done using some automated build process. In other words: if a build process exists which simply moves the appropriate images into place, updates the required styles, etc... then you could just run that build process for every state. Granted, I have no idea how the current build process works, but conceptually this is the simplest approach I can think of.

As you and Anne told us to do parameterizing for Css work so we had given an option of global.scss file which is used as a common css file, and yoy are okay with it so that's why we remove our page.scss code and move it to global.scss file, please let us know if  we need to change it again.


That's why removing page level css and add all css in global css file.

Reply:
Do _all_ styles need to be parametrized though? Would it be reasonable to leave the styles as-is for now, and only move the ones that need to be parametrized when you're doing the parametrization work?

Reply:
we can't use parametrization for css that's why we gave option for a common file global.scss file and you are okay with it so we move all the css file to global.scss file. As we need to do some change?

Reply:
That was admittedly an easier thing to agree to when there were less than 100 lines in that file. 

If you're now saying that every style in the application needs to be parametrizable and live at the global level (which is now >400 lines), that's a much different story.

Reply:
The decision was made that this is acceptable.